.node-version

```diff
@@ -0,0 +1 @@
+14.17.6
```

Comment:
you can use the `engines` field to specify the node version in react projects
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines

Reply:
specifying with a `.node-version` file looks fine as well, although I have never used it...
https://stackoverflow.com/questions/27425852/what-uses-respects-the-node-version-file